tests/api/test_extensions.py

```diff
@@ -2,74 +2,23 @@
 import pytest
 
 from saleor.extensions import ConfigurationTypeField
-from saleor.extensions.base_plugin import BasePlugin
 from saleor.extensions.manager import get_extensions_manager
 from saleor.extensions.models import PluginConfiguration
 from tests.api.utils import get_graphql_content
+from tests.extensions.helpers import get_config_value
+from tests.extensions.sample_plugins import PluginSample
 
 
-class PluginSample(BasePlugin):
-    PLUGIN_NAME = "PluginSample"
-    CONFIG_STRUCTURE = {
-        "Username": {
-            "type": ConfigurationTypeField.STRING,
-            "help_text": "Username input field",
-            "label": "Username",
-        },
-        "Password": {
-            "type": ConfigurationTypeField.STRING,
-            "help_text": "Password input field",
-            "label": "Password",
-        },
-        "Use sandbox": {
-            "type": ConfigurationTypeField.BOOLEAN,
-            "help_text": "Use sandbox",
-            "label": "Use sandbox",
-        },
-    }
-
-    @classmethod
-    def get_plugin_configuration(cls, queryset) -> "PluginConfiguration":
-        qs = queryset.filter(name="PluginSample")
-        if qs.exists():
-            return qs[0]
-        defaults = {
-            "name": "PluginSample",
-            "description": "Test plugin description",
-            "active": True,
-            "configuration": [
-                {
-                    "name": "Username",
-                    "value": "admin",
-                    "type": ConfigurationTypeField.STRING,
-                    "help_text": "Username input field",
-                    "label": "Username",
-                },
-                {
-                    "name": "Password",
-                    "value": "123",
-                    "type": ConfigurationTypeField.STRING,
-                    "help_text": "Password input field",
-                    "label": "Password",
-                },
-                {
-                    "name": "Use sandbox",
-                    "value": False,
-                    "type": ConfigurationTypeField.BOOLEAN,
-                    "help_text": "Use sandbox",
-                    "label": "Use sandbox",
-                },
-            ],
-        }
-        return PluginConfiguration.objects.create(**defaults)
+@pytest.fixture
+def staff_api_client_can_manage_plugins(staff_api_client, permission_manage_plugins):
```

Comment:
I'm not sure if we should introduce such fixture. 
In other tests, we add permission inside each test and I think we should have one approach in all tests for better readability.

Reply:
Readability is one thing, but I prefer leaving code better than I've found it :) I think that adding permissions at the beginning of the test is good only when we test permissions. In other cases, we can just add permissions in a fixture to omit additional lines ;)